pkg/universe.dagger.io/x/baptiste.barbotin@epitech.eu/azure/azure/functionApp/create.cue

```diff
@@ -0,0 +1,45 @@
+package functionApp
+
+import (
+	"universe.dagger.io/docker"
+)
+
+#Create: {
+
+	image: docker.#Image
+
+	// Azure FunctionApp version
+	version: string
+
+	// Function name
+	name: string
+
+	// Function location
+	location: string
+
+	// ResourceGroup name
+	resourceGroup: name: string
+
+	// Storage name
+	storage: name: string
+
+	// Additional arguments
+	args: [...string] | *[]
```

Comment:
You are embedding `docker.#Run` here, so this flags is useless since it's already define by `docker.#Run`